manager-dashboard/.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
```

Comment:
same question as above: where would be the best place for this ci.yml?

Reply:
These are for specific components (pulled when we merged another repo to this one). Maybe we can discuss further on this.